### ISSUE TITLE
Deploying Neo4j database using Docker has bugs during runtime

### ISSUE THREAD

**Issue by sybest1259** [neo4j/neo4j#13133]
System environment:
- Neo4j version: 5.3.0
- Neo4j docker: neo4j:5.3.0-community(https://hub.docker.com/_/neo4j)
- Operating system: Ubuntu 18.04

Problem:
When Neo4j Docker unexpectedly stops during the execution of Cypher or encounters an OOM problem causing Docker to crash, restarting Neo4j Docker will fail and there will be no logs in the Docker logs.

The command to create Neo4j Docker:
```
docker run -dit --name neo4j_prod \
-p 7474:7474 -p 7687:7687 \
--env NEO4J_AUTH=neo4j/s12345678 \
neo4j:5.3.0-community 
```

**Comment by ojhughes** [comment]
Hi, please ensure memory settings are configured with appropriate values

1. https://neo4j.com/docs/operations-manual/current/performance/memory-configuration/
2. https://neo4j.com/docs/operations-manual/current/docker/ref-settings/

**Comment by loveleif** [comment]
Also note, some issues with memory over-estimation has been solved since 5.3.0. So it might be worth to upgrade if you configure transaction memory limits (like `[db.memory.transaction.total.max](https://neo4j.com/docs/operations-manual/current/reference/configuration-settings/#config_db.memory.transaction.total.max)).
